package com.example.TaskManager.dto;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.Size;

public record CategoryRequest(
        @NotBlank(message = "Category name is required")
        @Size(min = 2, max = 50, message = "Category name must be between 2 and 50 characters")
        String categoryName,

        @Size(max = 500, message = "Description cannot exceed 500 characters")
        String categoryDescription
) {
}